cmd/edgectl/aes_install.go

```diff
@@ -605,79 +644,46 @@ func (i *Installer) Perform(kcontext string) error {
 		i.Report("cluster_not_accessible")
 		i.show.Println("-> Local cluster detected. Not configuring automatic TLS.")
 		i.show.Println()
-		i.ShowWrapped(noTlsSuccess)
+		i.ShowWrapped(color.Bold.Sprintf(noTlsSuccess))
 		i.show.Println()
-		loginMsg := "Determine the IP address and port number of your Ambassador service.\n"
-		loginMsg += "(e.g., minikube service -n ambassador ambassador)\n"
-		loginMsg += fmt.Sprintf(loginViaIP, "IP_ADDRESS:PORT")
+		loginMsg := "Determine the IP address and port number of your Ambassador service, e.g.\n"
+		loginMsg += color.Bold.Sprintf("$ minikube service -n ambassador ambassador)\n\n")
+		loginMsg += fmt.Sprintf(loginViaIP)
+		loginMsg += color.Bold.Sprintf("$ edgectl login -n ambassador IP_ADDRESS:PORT")
```

Comment:
There's a superfluous close-parenthesis in the `minikube` example.

Reply:
I'll add the TODO as requested and check the minikube example, thanks.